<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>2</width>
    <height>2</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TOF&gt;cam 660</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>2</width>
     <height>2</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButtonLogo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-image:url(:/logo/images/logo/logo_tablet.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelProduct">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255,255,255);
/*color:rgb(255, 253, 251);*/
color:rgb(0,0,0);</string>
      </property>
      <property name="text">
       <string>TOF&gt;cam 660</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelSystemInfo">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;GUI - application version&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;WF - firmware version&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;WID - wafer identification number&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;CID - chip identification number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255,255,255);
/*color:rgb(255, 253, 251);*/
color:rgb(0,0,0);</string>
      </property>
      <property name="text">
       <string> GUI 
 FW 
 WID 
 CID  </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionView"/>
   <addaction name="actionPlay"/>
   <addaction name="actionSettings"/>
   <addaction name="actionCalibration"/>
   <addaction name="actionTools"/>
   <addaction name="actionSave_Load"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionView">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/toolbar/images/toolbar/view.png</normaloff>
     <activeoff>:/toolbar/images/toolbar/view_hovered.png</activeoff>:/toolbar/images/toolbar/view.png</iconset>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/toolbar/images/toolbar/calibration.png</normaloff>
     <activeoff>:/toolbar/images/toolbar/calibration_hovered.png</activeoff>:/toolbar/images/toolbar/calibration.png</iconset>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTools">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/toolbar/images/toolbar/test.png</normaloff>
     <activeoff>:/toolbar/images/toolbar/test_hovered.png</activeoff>:/toolbar/images/toolbar/test.png</iconset>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/toolbar/images/toolbar/command.png</normaloff>
     <activeon>:/toolbar/images/toolbar/command_hovered.png</activeon>:/toolbar/images/toolbar/command.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/toolbar/images/toolbar/settings.png</normaloff>:/toolbar/images/toolbar/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSave_Load">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/toolbar/images/toolbar/save_load.png</normaloff>:/toolbar/images/toolbar/save_load.png</iconset>
   </property>
   <property name="text">
    <string>Save/Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/images.qrc"/>
 </resources>
 <connections/>
</ui>
